labs/criteria.md

```diff
@@ -1,6 +1,6 @@
 # arXivLabs Criteria
 
-_*arXiv Labs is temporarily on pause while arXiv is focused on stability and moving to the cloud. Please check back in six months.*_
+_arXiv Labs is back from vacation._ As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

Comment:
```suggestion
As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

Reply:
The NSF grand folks will likely not know it was offline for a bit. Why draw attention to it?

Reply:
Well, this isn't only for NSF. What if it just said "We are, once again, open for ..."? And take out "As of December 2022 ..."